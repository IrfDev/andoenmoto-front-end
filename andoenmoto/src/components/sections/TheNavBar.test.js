import { mount, createLocalVue } from '@vue/test-utils';
import TheNavBar from './TheNavBar.vue';
import Vuex from 'vuex';

const localVue = createLocalVue();
localVue.use(Vuex);

const store = new Vuex.Store({
  modules: {
    auth: {
      state: {
        authId: 'asdasdasda',
      }
    }
  }
})

describe('Logo test snapshot', () => {
  it('Logo are the same', () => {
    const cmp = mount(TheNavBar,{store, localVue});
    expect(cmp.element).toMatchSnapshot();
  });
});