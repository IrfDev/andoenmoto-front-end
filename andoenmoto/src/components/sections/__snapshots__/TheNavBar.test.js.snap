// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo test snapshot Logo are the same 1`] = `
<nav
  class="navbar d-flex justify-content-between"
>
  <a
    class="navbar-brand"
  >
    <router-link
      to="/"
    >
      <img
        alt=""
        class="d-inline-block align-top"
        height="30"
        id="logo-nav"
        src="/identity/logo.png"
        width="30"
      />
    </router-link>
  </a>
   
  <span
    class="navbar-text"
  >
    <router-link
      class="d-flex justify-content-flex-end"
      to="/profile/me"
    >
      <avatar
        class="nav-avatar"
        displayname="true"
        verticalalignment="true"
      />
    </router-link>
  </span>
</nav>
`;
